sources/features/calt.fea

```diff
@@ -0,0 +1,26 @@
+lookup calt_contextual {
+  @ahead = [abreve adieresis agrave amacron atilde 
+            b bar braceright bracketright brokenbar ebreve ecaron edieresis 
+            egrave emacron h h.alt hbar hcircumflex i iacute ibreve 
+            icircumflex idieresis igrave
+            ij imacron iogonek itilde j jcircumflex k l lacute lcaron ldot
+            lslash odieresis ograve omacron ordfeminine otilde paragraph
+            parenright question quotedblleft quotedblright quoteleft quoteright
+            T thorn ubreve udieresis ugrave umacron uni00B2 uni00B3 uni00B9
+            uni013C uni0180 uni0185 uni0196 uni019A uni01D0 uni01F0 uni0209
+            uni020B uni021F uni0234 uni0249 uni1E03 uni1E05 uni1E07 uni1E23
+            uni1E25 uni1E27 uni1E29 uni1E2B uni1E2D uni1E2F uni1E31 uni1E33
+            uni1E35 uni1E37 uni1E39 uni1E3B uni1E3D uni1EC9 uni1ECB uni1EFB
+            uni2047 uni2048 uni2C61 V W Y Yacute ydieresis];
+  sub [f f_f]' lookup chaining_substitutions @ahead;
+# #ifdef NOSMALLCAPS
+  sub Q' lookup chaining_substitutions [u v];
+# #else
+  sub Q' lookup chaining_substitutions [u u.sc v v.sc];
+  sub q.sc' lookup chaining_substitutions [u.sc v.sc];
+# #endif
+} calt_contextual;
+
+feature calt {
+      lookup calt_contextual;
+} calt;
```

Comment:
In features that has one lookup and the lookup is not referenced any where else, you can even use the shorter syntax:
```
feature calt {
  # lookup content
} calt;
```

and skip the named lookup, the end result will be the same.

Reply:
I did know this - I prefer the separate tables. Do you want me to edit them down?

Reply:
No, I can do this later.